projects/storefrontlib/cms-components/navigation/page-header/page-title.component.ts

```diff
@@ -26,9 +28,14 @@ export class PageTitleComponent implements OnInit {
   }
 
   private setTitle(): void {
-    this.title$ = this.pageMetaService.getMeta().pipe(
-      filter(isNotNullable),
-      map((meta) => (meta.heading || meta.title) ?? '')
+    this.title$ = combineLatest([
```

Comment:
as mentioned, do not couple 'any checkout' logic to this 'generic' page title

Reply:
put it back to what it was and use resolveHeading() in the checkout-page resolver
```suggestion
     this.title$ = this.pageMetaService.getMeta().pipe(
      filter(isNotNullable),
      map((meta) => (meta.heading || meta.title) ?? '')
```